### PR TITLE
post-merge updates: intro to ether

### PR DESCRIPTION
**Pull request by jmcook1186**
## Description

updates `developers/docs/intro-to-ether/index.md` for post-merge 

## Related Issue

#7075

### CODE REVIEW

File: src/content/developers/docs/intro-to-ethereum/index.md [hunk 1]
```diff
@@ -16,15 +16,9 @@ A blockchain is a public database that is updated and shared across many compute
...
+Ethereum uses a proof-of-stake based consensus mechanism. This means that anyone who wants to add new blocks to the chain must stake at least 32 ETH into the deposit contract and run validator software. They then have a chance of being randomly selected to propose blocks that other validators check and add to the blockchain. In this model there is usually only one chain, but network latency and dishoinest behaviour can cause multiple blocks to exist at the same position near the head of the chain. To resolve this, a fork-choice algorithm selects one canonical set of blocks. The blocks selected are the ones that form the heaviest possible chain, where 'heavy' refers to the number of validators that have endorsed the blocks (weighted by the ETH they have staked). There is a system of rewards and penalties that strongly incentivize participants to be honest and online as much as possible.
```
Comment (wackerow) [inline src/content/developers/docs/intro-to-ethereum/index.md:19]:
```suggestion
Ethereum uses a proof-of-stake based consensus mechanism. This means that anyone who wants to add new blocks to the chain must stake at least 32 ETH into the deposit contract and run validator software. They then have a chance of being randomly selected to propose blocks that other validators check and add to the blockchain. In this model there is usually only one chain, but network latency and dishonest behavior can cause multiple blocks to exist at the same position near the head of the chain. To resolve this, a fork-choice algorithm selects one canonical set of blocks. The blocks selected are the ones that form the heaviest possible chain, where 'heavy' refers to the number of validators that have endorsed the blocks (weighted by the ETH they have staked). There is a system of rewards and penalties that strongly incentivize participants to be honest and online as much as possible.
```

File: src/content/developers/docs/intro-to-ethereum/index.md [hunk 2]
```diff
@@ -34,6 +28,8 @@ Watch Anders explain hashes in blockchains:
 
 ## What is Ethereum? {#what-is-ethereum}
 
+Ethereum is a blockchain with a computer embedded in it. It serves as the foundation for building apps and organizations is a decentralized, permissionless, censorship resistant way.
```
Comment (wackerow) [inline src/content/developers/docs/intro-to-ethereum/index.md:31]:
```suggestion
Ethereum is a blockchain with a computer embedded in it. It serves as the foundation for building apps and organizations in a decentralized, permissionless, censorship resistant way.
```

File: src/content/developers/docs/intro-to-ethereum/index.md [hunk 3]
```diff
@@ -42,11 +38,13 @@ Cryptographic mechanisms ensure that once transactions are verified as valid and
...
+The amount of ETH paid corresponds to the time required to do the computation. These bounties also prevent malicious participants from intentionally clogging the network by requesting the execution of infinite computation or other resource-intensive scripts, as these participants must pay for computation time.
 
-The amount of ether paid corresponds to the time required to do the computation. These bounties also prevent malicious participants from intentionally clogging the network by requesting the execution of infinite computation or other resource-intensive scripts, as these participants must pay for computation time.
+ETH is also used to provide crypto-economic security to the network, in three main ways: 1) it is used as a means to reward validators who propose blocks or call out dishonest behaviour by other validators; 2) It is staked by validators, acting as collateral against dishionest behaviour - if validators attempt to misbehave their ETH can be destroyed; 3) it is used to weight 'votes' for newly proposed blocks, feeding into the fork-choice part of the consensus mechanism.
```
Comment (wackerow) [inline src/content/developers/docs/intro-to-ethereum/index.md:47]:
```suggestion
ETH is also used to provide crypto-economic security to the network, in three main ways: 1) it is used as a means to reward validators who propose blocks or call out dishonest behavior by other validators; 2) It is staked by validators, acting as collateral against dishonest behavior—if validators attempt to misbehave their ETH can be destroyed; 3) it is used to weight 'votes' for newly proposed blocks, feeding into the fork-choice part of the consensus mechanism.
```

**Review by wackerow — APPROVED** [review]
@jmcook1186 LGTM! Just committed a couple tiny typo fixes

File: src/content/developers/docs/intro-to-ethereum/index.md [hunk 4]
```diff
@@ -16,15 +16,9 @@ A blockchain is a public database that is updated and shared across many compute
...
+Ethereum uses a proof-of-stake based consensus mechanism. This means that anyone who wants to add new blocks to the chain must stake at least 32 ETH into the deposit contract and run validator software. They then have a chance of being randomly selected to propose blocks that other validators check and add to the blockchain. In this model there is usually only one chain, but network latency and dishonest behavior can cause multiple blocks to exist at the same position near the head of the chain. To resolve this, a fork-choice algorithm selects one canonical set of blocks. The blocks selected are the ones that form the heaviest possible chain, where 'heavy' refers to the number of validators that have endorsed the blocks (weighted by the ETH they have staked). There is a system of rewards and penalties that strongly incentivize participants to be honest and online as much as possible.
```
Comment (minimalsm) [inline src/content/developers/docs/intro-to-ethereum/index.md:19]:
```suggestion
Ethereum uses a [proof-of-stake-based consensus mechanism](/developers/docs/consensus-mechanisms/pos). Anyone who wants to add new blocks to the chain must stake at least 32 ETH into the deposit contract and run validator software. They then can be randomly selected to propose blocks that other validators check and add to the blockchain. In this model, there is usually only one chain, but network latency and dishonest behavior can cause multiple blocks to exist at the same position near the head of the chain. To resolve this, a fork-choice algorithm selects one canonical set of blocks. The blocks selected are the ones that form the heaviest possible chain, where 'heavy' refers to the number of validators that have endorsed the blocks (weighted by the ETH they have staked). There is a system of rewards and penalties that strongly incentivize participants to be honest and online as much as possible.
```

File: src/content/developers/docs/intro-to-ethereum/index.md [hunk 5]
```diff
@@ -16,15 +16,9 @@ A blockchain is a public database that is updated and shared across many compute
...
 
-New blocks are broadcast to the nodes in the network, checked and verified, thus updating the state of the blockchain for everyone.
-
-So to summarize, when you send ETH to someone, the transaction must be added to the block which will get mined. The updated state is then shared with the entire network.
-
-Watch Austin walk you through blockchains:
-
-<YouTube id="zcX7OJ-L8XQ" />
+More on [proof-of-stake](/developers/docs/consensus-mechanisms/pos).
```
Comment (minimalsm) [inline src/content/developers/docs/intro-to-ethereum/index.md:21]:
```suggestion
```

File: src/content/developers/docs/intro-to-ethereum/index.md [hunk 6]
```diff
@@ -34,6 +28,8 @@ Watch Anders explain hashes in blockchains:
 
 ## What is Ethereum? {#what-is-ethereum}
 
+Ethereum is a blockchain with a computer embedded in it. It serves as the foundation for building apps and organizations in a decentralized, permissionless, censorship resistant way.
```
Comment (minimalsm) [inline src/content/developers/docs/intro-to-ethereum/index.md:31]:
```suggestion
Ethereum is a blockchain with a computer embedded in it. It is the foundation for building apps and organizations in a decentralized, permissionless, censorship-resistant way.
```

File: src/content/developers/docs/intro-to-ethereum/index.md [hunk 7]
```diff
@@ -42,11 +38,13 @@ Cryptographic mechanisms ensure that once transactions are verified as valid and
...
+The amount of ETH paid corresponds to the time required to do the computation. These bounties also prevent malicious participants from intentionally clogging the network by requesting the execution of infinite computation or other resource-intensive scripts, as these participants must pay for computation time.
 
-The amount of ether paid corresponds to the time required to do the computation. These bounties also prevent malicious participants from intentionally clogging the network by requesting the execution of infinite computation or other resource-intensive scripts, as these participants must pay for computation time.
+ETH is also used to provide crypto-economic security to the network, in three main ways: 1) it is used as a means to reward validators who propose blocks or call out dishonest behavior by other validators; 2) It is staked by validators, acting as collateral against dishonest behavior—if validators attempt to misbehave their ETH can be destroyed; 3) it is used to weight 'votes' for newly proposed blocks, feeding into the fork-choice part of the consensus mechanism.
```
Comment (minimalsm) [inline src/content/developers/docs/intro-to-ethereum/index.md:47]:
```suggestion
ETH is also used to provide crypto-economic security to the network in three main ways: 1) it is used as a means to reward validators who propose blocks or call out dishonest behavior by other validators; 2) It is staked by validators, acting as collateral against dishonest behavior—if validators attempt to misbehave their ETH can be destroyed; 3) it is used to weight 'votes' for newly proposed blocks, feeding into the fork-choice part of the consensus mechanism.
```

File: src/content/developers/docs/intro-to-ethereum/index.md [hunk 8]
```diff
@@ -42,11 +38,13 @@ Cryptographic mechanisms ensure that once transactions are verified as valid and
 
 ## What is ether? {#what-is-ether}
 
-**Ether (ETH)** is the native cryptocurrency of Ethereum. The purpose of ether is to allow for a market for computation. Such a market provides an economic incentive for participants to verify and execute transaction requests and provide computational resources to the network.
+**Ether (ETH)** is the native cryptocurrency of Ethereum. The purpose of ETH is to allow for a market for computation. Such a market provides an economic incentive for participants to verify and execute transaction requests and provide computational resources to the network.
+
+Any participant who broadcasts a transaction request must also offer some amount of ETH to the network as a bounty. This bounty will be awarded to whoever eventually does the work of verifying the transaction, executing it, committing it to the blockchain, and broadcasting it to the network.
```
Comment (minimalsm) [inline src/content/developers/docs/intro-to-ethereum/index.md:43]:
```suggestion
Any participant who broadcasts a transaction request must also offer some amount of ETH to the network as a bounty. The network will award this bounty to whoever eventually does the work of verifying the transaction, executing it, committing it to the blockchain, and broadcasting it to the network.
```

**Review by minimalsm — CHANGES_REQUESTED** [review]

File: src/content/developers/docs/intro-to-ethereum/index.md [hunk 9]
```diff
@@ -16,15 +16,8 @@ A blockchain is a public database that is updated and shared across many compute
...
+Ethereum uses a [proof-of-stake-based consensus mechanism](/developers/docs/consensus-mechanisms/pos). Anyone who wants to add new blocks to the chain must stake at least 32 ETH into the deposit contract and run validator software. They then can be randomly selected to propose blocks that other validators check and add to the blockchain. In this model, there is usually only one chain, but network latency and dishonest behavior can cause multiple blocks to exist at the same position near the head of the chain. To resolve this, a fork-choice algorithm selects one canonical set of blocks. The blocks selected are the ones that form the heaviest possible chain, where 'heavy' refers to the number of validators that have endorsed the blocks (weighted by the ETH they have staked). There is a system of rewards and penalties that strongly incentivize participants to be honest and online as much as possible.
```
Comment (minimalsm) [inline src/content/developers/docs/intro-to-ethereum/index.md:19]:
```suggestion
Ethereum uses a [proof-of-stake-based consensus mechanism](/developers/docs/consensus-mechanisms/pos/). Anyone who wants to add new blocks to the chain must stake at least 32 ETH into the deposit contract and run validator software. They then can be randomly selected to propose blocks that other validators check and add to the blockchain. In this model, there is usually only one chain, but network latency and dishonest behavior can cause multiple blocks to exist at the same position near the head of the chain. To resolve this, a fork-choice algorithm selects one canonical set of blocks. The blocks selected are the ones that form the heaviest possible chain, where 'heavy' refers to the number of validators that have endorsed the blocks (weighted by the ETH they have staked). There is a system of rewards and penalties that strongly incentivize participants to be honest and online as much as possible.
```

**Review by minimalsm — APPROVED** [review]
Nice :-)

I think we should remove the videos (at least Austin's, which heavily refers to proof-of-work). Happy to split that discussion off into a more general 'what do we do with external resources' discussion at some point soon. [I've made a Notion card here to capture that work.](https://www.notion.so/efdn/Evaluate-external-resources-used-on-ethereum-org-15fffa1bc7ff45eaacfe0885b53245fb)

File: src/content/developers/docs/intro-to-ethereum/index.md [hunk 10]
```diff
@@ -42,11 +37,13 @@ Cryptographic mechanisms ensure that once transactions are verified as valid and
...
+The amount of ETH paid corresponds to the time required to do the computation. These bounties also prevent malicious participants from intentionally clogging the network by requesting the execution of infinite computation or other resource-intensive scripts, as these participants must pay for computation time.
 
-The amount of ether paid corresponds to the time required to do the computation. These bounties also prevent malicious participants from intentionally clogging the network by requesting the execution of infinite computation or other resource-intensive scripts, as these participants must pay for computation time.
+ETH is also used to provide crypto-economic security to the network in three main ways: 1) it is used as a means to reward validators who propose blocks or call out dishonest behavior by other validators; 2) It is staked by validators, acting as collateral against dishonest behavior—if validators attempt to misbehave their ETH can be destroyed; 3) it is used to weight 'votes' for newly proposed blocks, feeding into the fork-choice part of the consensus mechanism.
```
Comment (minimalsm) [inline src/content/developers/docs/intro-to-ethereum/index.md:46]: Let's use an actual ordered list (`<ol> <li>`) here instead of listing in text.

**Pull request merged**